### ISSUE TITLE
how do I use riv file？

### ISSUE THREAD

**Issue by ModWU** [rive-app/rive-flutter#99]
If I get an RIV file from Community, how do I use it？I have learned the demo in the library, but I still don't know how to use other people's RIV files. Is there a detailed tutorial for using it?

**Comment by ModWU** [comment]
> 
> 
> If I get an RIV file from Community, how do I use it？I have learned the demo in the library, but I still don't know how to use other people's RIV files. Is there a detailed tutorial for using it?

![图片](https://user-images.githubusercontent.com/16234264/118217489-24cf4180-b4a8-11eb-9133-c831ac6283aa.png)

If I have this information in the figure above, how can I write code to use it?

**Comment by mjohnsullivan** [comment]
There's a few examples in the Github repo that should help you get started: https://github.com/rive-app/rive-flutter/tree/master/example

For state machines, take a look at the `Skills Machine` example; it shows how to get a state machine's inputs and modify them: https://github.com/rive-app/rive-flutter/blob/master/example/lib/state_machine_skills.dart

We'll be producing more examples and tutorials over the coming weeks.

**Comment by mjohnsullivan** [comment]
I agree it should be more obvious how to get state machine and animation names. I'll open up a ticket to fix that up.

**Comment by mjohnsullivan** [comment]
See #104
